Reverse first k elements of queue.cpp

```diff
@@ -0,0 +1,24 @@
+/*
+Please note that it's Function problem i.e.
+you need to write your solution in the form of Function(s) only.
+Driver Code to call/invoke your function would be added by GfG's Online Judge.*/
+
+
+//User function Template for C++
+queue<ll> modifyQueue(queue<ll> q, int k)
```

Comment:
When writing a function, make sure the inputs being passed are valid first before getting into business logic.